source/fundamentals/linq.txt

```diff
@@ -0,0 +1,541 @@
+    { "name" : "Isle Of Capri Resturant", "cuisine" : "Italian" }
+    { "name" : "Marchis Restaurant", "cuisine" : "Italian" }
+    { "name" : "Crystal Room", "cuisine" : "Italian" }
+    { "name" : "Forlinis Restaurant", "cuisine" : "Italian" }
+
+$skip
+~~~~~
+
+The ``$skip`` aggregation stage skips over a specified number of documents returned
+by a query, then returns the rest of the results. The following example shows how to generate
+a ``$skip`` stage using LINQ:
+
+.. code-block:: csharp
+   :emphasize-lines: 4
+
+   var query = queryableCollection
+       .Where(r => r.Cuisine == "Italian")
+       .Select(r => new {r.Name, r.Cuisine})
+       .Skip(2);
+
+The preceding example skips the first two restaurants that match the criteria, and
+returns the rest as follows:
+
+.. code-block:: json
+
+   // Results Truncated
+
+   { "name" : "Marchis Restaurant", "cuisine" : "Italian" }
+   { "name" : "Crystal Room", "cuisine" : "Italian" }
+   { "name" : "Forlinis Restaurant", "cuisine" : "Italian" }
+   ...
+
+$unwind
+~~~~~~~
+
+The ``$unwind`` aggregation stage deconstructs a specified array field and returns
+a document for each element in that array.
+
+Select the :guilabel:`Query Syntax` or :guilabel:`Method Syntax` tab to see how 
+to generate an ``$unwind`` stage using LINQ:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 3
+
+         var query = from r in queryableCollection
+             where r.Name == "The Movable Feast"
+             from grade in r.Grades
+             select grade;
+
+   .. tab:: Method Syntax
+      :tabid: method-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 3
+
+         var query = queryableCollection
+             .Where(r => r.Name == "The Movable Feast")
+             .SelectMany(r => r.Grades);
+
+The query in the preceding example finds the document where the ``Name`` field 
+has the value "The Movable Feast." Then, for each element in this document's 
+``Grades`` array, the query returns a new document. The results are as follows:
+
+.. code-block:: json
+
+   { "date" : ISODate("2014-11-19T00:00:00Z"), "grade" : "A", "score" : 11 }
+   { "date" : ISODate("2013-11-14T00:00:00Z"), "grade" : "A", "score" : 2 }
+   { "date" : ISODate("2012-12-05T00:00:00Z"), "grade" : "A", "score" : 13 }
+   { "date" : ISODate("2012-05-17T00:00:00Z"), "grade" : "A", "score" : 11 }
+
+$group
+~~~~~~
+
+The ``$group`` aggregation stage separates documents into groups according to
+the criteria you specify.
+
+Select the :guilabel:`Query Syntax` or :guilabel:`Method Syntax` tab to see how 
+to generate an ``$group`` stage using LINQ:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      .. code-block:: csharp
+         :emphasize-lines: 2
+
+         var query = from r in queryableCollection
+             group r by r.Cuisine into g
+             select new {Cuisine = g.Key, Count = g.Count()};
```

Comment:
I agree that query syntax is uglier/more difficult to use and I personally would shy away from it too. I mainly included it here to ensure we covered it since it's in the original documentation. 

@JamesKovacs what do you think on query syntax being shown on these docs? Should we keep it but switch the tab order so that method syntax is shown first? Or omit it altogether?

Reply:
Valid points. Query comprehension syntax is common enough in C# that I think we should include it in our docs, but let's re-order the tabs so that it doesn't appear first.

As for testing the equivalency between the two syntaxes, I would suggest we do the obvious translation - which can often be done via an automatic Rider refactoring - and leave it at that. If someone reports that one of our method chained versions produces different MQL than our query comprehension version, then we can investigate and amend at that time.